src/ios/CDVStatusBar.m

```diff
@@ -200,8 +202,8 @@ - (void) initializeStatusBarBackgroundView
 
 - (CGRect) invertFrameIfNeeded:(CGRect)rect {
     // landscape is where (width > height). On iOS < 8, we need to invert since frames are
-    // always in Portrait context
-    if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation]) && (rect.size.width < rect.size.height)) {
+    // always in Portrait context. Do not run this on ios 8 or above to avoid breaking ipad pro multitask layout
+    if (!IsAtLeastiOSVersion(@"8.0") && UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation]) && (rect.size.width < rect.size.height)) {
```

Comment:
You can remove the `rect.size.width < rect.size.height` part as that will only be true on iOS 7 and earlier and you already added the `!IsAtLeastiOSVersion(@"8.0")` check